.gitignore

```diff
@@ -2,6 +2,7 @@
 .env
 /dist
 /node_modules
```

Comment:
You should have just remove the node_modules but not the nonce-signer. Just adjust this line to node_modules